go/client/cmd_simplefs_read.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"fmt"
+	"io"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+// CmdSimpleFSRead is the 'simplefs read' command.
+type CmdSimpleFSRead struct {
+	libkb.Contextified
+	path keybase1.Path
+}
+
+// NewCmdDeviceList creates a new cli.Command.
+func NewCmdSimpleFSRead(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "read",
+		ArgumentHelp: "<path>",
+		Usage:        "output file contents",
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(&CmdDeviceList{Contextified: libkb.NewContextified(g)}, "read", c)
+		},
+	}
+}
+
+// Run runs the command in client/server mode.
+func (c *CmdSimpleFSRead) Run() error {
+	cli, err := GetSimpleFSClient(c.G())
+	if err != nil {
+		return err
+	}
+
+	opid, err := cli.SimpleFSMakeOpid(context.TODO())
+	if err != nil {
+		return err
+	}
+	err = cli.SimpleFSOpen(context.TODO(), keybase1.SimpleFSOpenArg{
+		OpID:  opid,
+		Dest:  c.path,
+		Flags: keybase1.OpenFlags_READ,
+	})
+	defer cli.SimpleFSClose(context.TODO(), opid)
+	if err != nil {
+		return err
+	}
+	var offset = 0
+	for {
+		content, err := cli.SimpleFSRead(context.TODO(), keybase1.SimpleFSReadArg{
+			OpID:   opid,
+			Offset: offset,
+			Size:   1024,
```

Comment:
should use a way bigger buffer size.  Try `16k`.  Should be tunable via input parameter too